src/generated/resources/data/immersiveengineering/recipes/crusher/glowstone.json

```diff
@@ -9,4 +9,4 @@
     "item": "minecraft:glowstone"
   },
   "energy": 3200
-}
\ No newline at end of file
+}
```

Comment:
You can't just rename this file. Datageneration is done programatically. You need to adjust the datagen code.